src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/seed/cas1/ApprovedPremisesRoomsSeedFromXLSXJob.kt

```diff
@@ -83,13 +87,13 @@ class ApprovedPremisesRoomsSeedFromXLSXJob(
     val beds = mutableListOf<BedEntity>()
     for (i in 1..<dataFrame.columnsCount()) {
       val roomAnswers = dataFrame.getColumn(i)
-      val bedNumber = roomAnswers[1].toString()
+      val roomCode = "$qCode - ${roomAnswers[0]}"
```

Comment:
may be worth introducing a `fun buildRoomCode(qCode,roomNumber) = "$qCode - $roomNumber"` to reduce duplication